lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -878,6 +907,11 @@ def cmd_vulns(*args)
       end
     end
 
+    unless tbl.rows.empty?
```

Comment:
IIRC, the functionality of `hosts`, `services`, `notes` and others is to produce output to the file regardless of what is found, in this case just the headers for the fields in question.  IMO consistency is best for now.

Reply:
Sounds good. I will do that. Thanks for the feedback!

Reply:
And done!